### ISSUE TITLE
[Bug][a11y]: Separator in split-view is not visible in high contrast mode

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3381]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Split-view

### Expected behavior

The separator line must be visible in high contrast mode. Also missing is a focus indicator in when the separator gets keyboard focus.

Use this [entry](https://accessibility.corp.adobe.com/docs/visual_design/high_contrast_mode/) to find an appropriate colors for default and focused states to show in high contrast mode.

### Actual behavior

Separator is not visible in windows high contrast mode. Nor is the focused state when it receives keyboard focus

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/split-view/
2. Simulate high contrast mode on browser using forced-colors rendering option in the browser
3. Observe that the control is not visible and focus states are not visible

**Comment by najikahalsema** [comment]
Does this fall into the DIY tokens category of work? @pfulton @jnjosh
